src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,19 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+  status: 'Under construction',
+};
+
+const categoriesSlice = createSlice({
+  name: 'category',
+  initialState,
+  reducers: {
+    checkStatus: (state) => {
+      state.status = 'Under construction';
+    },
```

Comment:
- [Optional] Dear @lily4178993, every Redux reducer method needs to take a `state` and `action`. these two are the common parameter of a reducer, without these parameters the reducer will not be useful in any way, Please consider adding the `action` parameter to your `checkStatus` reducer method.

Reply:
Hi @TadesseDev.
In Redux Toolkit, when using the createSlice function, it is not necessary to add an action to the reducer manually. Redux Toolkit takes care of generating the action creators for you automatically.

However, I take your suggestion into account and modify the initial code.